jb-lib/pyproject.toml

```diff
@@ -16,7 +16,7 @@ pydantic = "^2.5.3"
 pydantic-settings = "^2.1.0"
 alembic = "^1.13.1"
 psycopg2-binary = "^2.9.9"
-aiohttp = "^3.10.0"
+aiohttp = "3.9.5"
```

Comment:
Is package downgrade required?

Reply:
Yes it conflicts with r2r package, so the downgrade was needed to 3.9.5.

Reply:
@KaranrajM Can you point to the r2r package? The current one on the github does not depend on aiohttp.